### PR TITLE
XML changes

### PR DESCRIPTION
**Pull request by jyucra**
* Sorted blocks out into the correct categories for the scenarios were this was still missing
* Made changes to the available blocks in Mission 3 so that you will only find a block in your toolbox if you will need it to complete that scenario.

### CODE REVIEW

**Comment by AmbientOSX** [comment]
Set mission 3 task 7 procedure definition to be collapsed (in the xml set collapsed="true") then :+1:

**Comment by jyucra** [comment]
I've rebased the branch to solve conflicts and made a couple extra changes to account for the separation of the conditional blocks. Another look would be appreciated.

File: source/scenario/newmission3task4.vwf.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ properties:
...
   - loadToolbox:
     - rover
-    - source/scenario/blockly/mission3b.xml
+    - source/scenario/blockly/mission3d.xml
```
Comment (kadst43) [inline source/scenario/newmission3task4.vwf.yaml]: In ideal world, we'd name the files consistently as MissionXTaskY (no "new", no "mission3x")...

**Comment by kadst43** [comment]
I added a small, nitpicky comment that doesn't need to be part of this PR... but in the meantime, let's get these changes in so that we can start playing with them.

**Pull request merged**